src/routes/+layout.svelte

```diff
@@ -164,96 +173,119 @@
 			home: {
 				icon: {
 					component: Home,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: '/'
 			},
 
 			discover: {
 				icon: {
 					component: Explore,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: undefined
 			},
 			list: {
 				icon: {
 					component: List,
-					style: "width: 1.7vw",
+					style: 'width: 1.7vw',
 					color: 'white'
-				}
+				},
+				url: undefined
 			},
 			schedule: {
 				icon: {
 					component: Schedule,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: undefined
 			},
 			forum: {
 				icon: {
 					component: Forum,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: undefined
 			}
 		},
 		bottom: {
 			settings: {
 				icon: {
 					component: Settings,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: undefined
 			},
 			'misc.': {
 				icon: {
 					component: Misc,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
-				}
+				},
+				url: undefined
 			}
 		},
 		profile_dropdown: {
 			profile: {
 				name: 'Profile',
-				link: '/profile/',
+				url: '/profile/',
 				icon: {
 					component: User,
-					style: "width: 1.25vw;",
+					style: 'width: 1.25vw;',
 					color: 'white'
 				}
 			},
 			my_list: {
 				name: 'My List',
-				link: '/mylist/',
+				url: '/mylist/',
 				icon: {
 					component: List,
-					style: "width: 1.5vw;",
+					style: 'width: 1.5vw;',
 					color: 'white'
 				}
 			},
 			theme: {
 				name: 'Theme',
-				link: '/theme/',
+				url: '/theme/',
 				icon: {
 					component: Moon,
-					style: "width: 1.1vw;",
+					style: 'width: 1.1vw;',
 					color: 'white'
 				}
 			},
 			settings: {
 				name: 'Settings',
-				link: '/settings/',
+				url: '/settings/',
 				icon: {
 					component: SettingsOutline,
-					style: "width: 1.1vw;",
+					style: 'width: 1.1vw;',
 					color: 'white'
 				}
 			}
 		}
 	};
 
+	// Activate button based on Urls
+	$: {
+		Object.values(icon_mapping).forEach((_) => {
+			Object.entries(_).forEach((item) => {
+				const button_name = item[0] as typeof active_button;
+				const internal_object = item[1];
+				if ('url' in internal_object) {
+					const url = internal_object.url as String;
+					if (url === $page.url.pathname) {
+						active_button = button_name;
+					}
```

Comment:
url has "/" at start and end, does $path.url.pathname returns with that "/" too?
Its a  doubt

Reply:
it does :)